bin/install_dependencies

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+BASEDIR=`dirname $0`/..
+
+apt-get update
+apt-get build-dep -y python-imaging
+apt-get install -y python-virtualenv libjpeg8 libjpeg62-dev libfreetype6 libfreetype6-dev npm
+cd $BASEDIR/apps/volontulo && npm install --global gulp-cli
```

Comment:
There is no need to go to `apps/volontulo` to execute `npm install gulp-cli`. The only reason to go there is if You're going to use `package.json`, which is placed there.